pkg/cstor/cvr_operation.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2019 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cstor
+
+import (
+	"github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// waitForAllCVR will ensure that all CVR related to
+// given volumes are created
+func (p *Plugin) waitForAllCVR(vol *Volume) error {
```

Comment:
should we have a timelimit or iterations limit here?
Also, seeing tight loop which can be fixed

Reply:
Add a log as well to see what is happening

Reply:
Done